examples/models/logistic_regression.py

```diff
@@ -7,5 +7,5 @@ def __init__(self, input_dim: int, output_dim: int) -> None:
         self.linear = torch.nn.Linear(input_dim, output_dim)
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
-        outputs = torch.sigmoid(self.linear(x)).squeeze()
+        outputs = torch.sigmoid(self.linear(x)).reshape(-1)
```

Comment:
If the batch size is 1 (which can happen at the end of a dataloader if the len(dataloader) mod batch_size = 1) then squeeze will cause problems in the BCE loss function calculation. Using reshape rather than squeeze avoids this.